app/screens/staking-unbonding-delegation-amount-input-screen/staking-unbonding-delegation-amount-input-screen.tsx

```diff
@@ -64,7 +66,9 @@ export class StakingUnbondingDelegationAmountInputScreen extends React.Component
   }
 
   private onPressNextButton = async () => {
+    logAnalyticsEvent('DelegateClickUnbound')
     if (await this.createTransactionForSigning()) {
+      logAnalyticsEvent('DelegateCreateUnboundTx')
```

Comment:
`StakingUndelegateCreateTxForSigning`

Reply:
Can we somehow make this event name (rule) shorter, this will hit 40 char if i later add some prefix or suffix sosad

Reply:
`StakeUndelegateCreateTxForSign`